source/util/AMDSErrorMonitor.h

```diff
@@ -0,0 +1,275 @@
+/*
```

Comment:
Is there a reason why you can't just use AMErrorMonitor?  I know that you've needed to have your own AMDSAxisInfo and what not because there were changes that needed to be made, but what is the necessity of making another error monitor?